### PR TITLE
Add notice that LFS mirroring is not supported

### PR DESCRIPTION
**Pull request by fabian-z**
As discussed in #849:

Mirroring LFS objects is no easy feat, so this PR adds a notice to the migration UI which tells the user that Gitea does not support it currently, hinting at ```git lfs``` functionality.

### CODE REVIEW

**Comment by pgaskin** [comment]
LGTM

File: options/locale/locale_de-DE.ini [hunk 1]
```diff
@@ -431,6 +431,7 @@ migrate.clone_local_path=oder Pfad auf dem lokalen Server
 migrate.permission_denied=Ihnen fehlen die Rechte zum Importieren lokaler Repositories.
 migrate.invalid_local_path=Der lokale Pfad ist ungültig, existiert nicht oder ist kein Ordner.
 migrate.failed=Fehler bei Migration: %v
+migrate.lfs_mirror_unsupported = Migrieren von LFS Objekten wird nicht unterstützt - verwenden Sie stattdessen 'git lfs fetch --all' und 'git lfs push --all'.
```
Comment (andreynering) [inline options/locale/locale_de-DE.ini]: Strings for langs other than English should not be added directly, but via [Crowdin](https://crowdin.com/project/gitea)

Comment (lunny) [reply options/locale/locale_de-DE.ini]: ^

**Review by lunny — CHANGES_REQUESTED** [review]

**Comment by fabian-z** [comment]
Dropped German translation as requested

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
LGTM

**Pull request merged**